### DISCUSSION TITLE
Corsair H100i V2 Temperature

### DISCUSSION THREAD

**Discussion by rafatuners** [Rem0o/FanControl.Releases#2588] [General]
Hello.

I have an old Corsair H100i V2 and in ICUE it shows the cooler temperature in addition to the processor temperature. I would like to use this cooler temperature as a reference. I wanted to know if there is a way to make the fan control monitor this sensor?

**Comment by Fr0stX76** [comment]
Have you tried this plugin?

https://github.com/EvanMulawski/FanControl.CorsairLink
